Examples/UIExplorer/js/NativeAnimationsExample.js

```diff
@@ -45,17 +45,23 @@ class Tester extends React.Component {
   current = 0;
 
   onPress = () => {
-    const animConfig = (
-      this.current && this.props.reverseConfig ? this.props.reverseConfig : this.props.config
-    );
+    const animConfig = this.current && this.props.reverseConfig
```

Comment:
Sorry about the noise in this file, ran prettier because I lost my ability for format JS properly :)